dpdd.tex

```diff
@@ -763,8 +763,6 @@ \subsubsection{Object Characterization Measures}
 
     \item \emph{Adaptive moments}. Adaptive moments will be computed using information from all epochs, independently for each band. The moments of the PSF realized at the position of the object will be provided as well. \dmreq{0276}
 
-    \item \emph{Local shear estimation measures}. Per-object quantities that can be used to construct de-biased estimates of the average shear of a population of galaxies (i.e. for weak gravitational lensing) will be provided.  Algorithms being considered for this role include \texttt{Metacalibration} \citep{2017arXiv170202600H,2017ApJ...841...24S} and the ``Bayesian Fourier-Domain'' approach of \citet{2016MNRAS.459.4467B}, and implementations are expected to rely heavily on existing third-party code.  The actual algorithm selection will be made in consultation with LSST science collaborations and will favor mature algorithms that have been used successfully on precursor surveys, but may also be constrained by compute resources and costs.  The exact nature of the quantities produced is algorithm-dependent, and hence is not included in Table~\ref{tbl:objectTable}.
```

Comment:
I am not sure if we need to remove this paragraph in its entirety since there may still be other algorithms (HSM, FPFS etc.) that would report per-object shear estimates.

Reply:
HSM is unlikely to be seriously used. FPFS will have its own object detections, just like metadetection. Thus they would not be reported for objects in the main table.

Reply:
My understanding is that we run only one shear estimation algorithm as part of the data release processing to produce per-object quantities in a formal LSST data release,  and that the DESC want the algorithm to be metadetection.  Running FPFS sounds like it would entail another new table?

Reply:
I see this bulleted list as being placeholders for things that are not more fully described elsewhere.  I do reference also running other algorithms ater (with the _huge_ caveat that they can't involve any nontrivial compute or storage costs, as I think might still leave room for BFD or something similar).

If FPFS would require another distinct Object table, I think that rules it out for the early DRs at least.

Reply:
Given this, I think we can remove this paragraph and handle any future changes to algorithms or additional algorithms as part of "Additional algorithms may be run as well .."

Reply:
Yes!! FPFS has different detections; therefore require another distinct ojbect table !!